src/TulsiGenerator/Bazel/tulsi/tulsi_aspects.bzl

```diff
@@ -896,6 +896,13 @@ def _tulsi_sources_aspect(target, ctx):
 
     # Keys for attribute and inheritable_attributes keys must be kept in sync
     # with defines in Tulsi's RuleEntry.
+
+    # FIXME(rules_ios)
```

Comment:
Would suggest just adding the rest while we're at it:
```
if kind == "apple_framework_packaging" && getattr("link_dynamic):
    is_dynamic = True
elif kind == "ios_framework"
   is_dynamic = True
```

Reply:
Done